docs/packaging.rst

```diff
@@ -225,32 +225,77 @@ src/snek5000_canonical/Snakefile
 
 .. code-block:: python
 
```

Comment:
Maybe we can add
```
from snakemake.utils import min_version
min_version("6.0")
```
?

Reply:
It is already set in `setup.cfg`, but sure why not :)

Reply:
Yes, finally it may not be so necessary. I was thinking about a user using this snakefile in another environment but anyway, it's going to crash soon when importing snek5000.